modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/ManageCollaboratorsMVCRenderCommand.java

```diff
@@ -34,14 +39,28 @@ public String render(
 			RenderRequest renderRequest, RenderResponse renderResponse)
 		throws PortletException {
 
-		if (!CTPermission.contains(
-				PermissionThreadLocal.getPermissionChecker(),
```

Comment:
Earlier I was checking the INVITE_USERS permission with this CTPermission but it was incorrect and was checking `com.liferay.change.tracking#INVITE_USER` which yielded an error so I had to update the check to CTCollectionPermission